docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1965 @@
+{
```

Comment:
My first version did not have `for-else` . My first version had `and min_dist >= 1e-6` condition in `while-loop conditional expression`.
```
while (

len(discords_idx) < k

and np.any(include)

and min_dist >= 1e-6  # config.STUMPY_MIN_DISCORD_DISTANCE

):
```
However, I had to use `for-else` as I was trying to avoid the following scenario (I noticed it when I was trying different values for decay)

Let us assume that the two last elements of `min_distances` are [...., 11, 1e-6]. (btw, this can easily happen if decay is not small (e.g. 0.2 instead of 0.01)) 

Then, after using `min_dist=11` and find no discords, the next choice for `min_dist` is `1e-6`. And, the while-loop will be run infinite times if it finds no discord again! One idea to resolve this might be using the condition `min_dist > 1e-6` (I removed equal sign). 
However, there is another problem: if `min_dist` becomes `1e-6` inside the `while-loop` and we return only 1 discord from set of candidates (not all possible discords), in the next iteration, the `while-loop` condition becomes false as `min_dist` is now 1e-6. 

---

From now on, as you said before, I will try to avoid covering all bad (extreme?) scenarios. I will provide some notes in the notebook if I feel we should take care of edge cases in the future. And, I will go with the cleaner implementation.